components/index/CardsMonitoring/PositiveRate/Chart.vue

```diff
@@ -102,7 +102,7 @@
 <script lang="ts">
 import { ChartOptions, PluginServiceRegistrationOptions } from 'chart.js'
 import dayjs from 'dayjs'
-import Vue from 'vue'
+import Vue, {PropType} from 'vue'
```

Comment:
```suggestion
import Vue, { PropType } from 'vue' // eslint-disable-line import/named
```

おっと、eslintエラーが出てしまいました。こちら↑のようにしてもらっておいていいでしょうか？

Reply:
お願いします！